pkg/didcomm/transport/mock/mock_interface.go

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
@rolsonquadras rename this to `mock_interface_test.go` to exclude it from code coverage

Reply:
@llorllale valid point. I'll update this.

Reply:
@llorllale i have updated the file name to mock_transport.go as it is more appropriate than mock_interface.go. If there are no test files, then it doesn't affect the coverage. And having only _test.go file gives error **" no non-test Go files in"** (this error make sense for package testing as there is no actual code).

Reply:
@rolsonquadras perhaps it should be moved to `internal` then. Leaving it like this means we'll be shipping it along with exported production code. Moving it to `internal` would at least render it unavailable to user code.

Reply:
@llorllale as suggested, moved to internal https://github.com/hyperledger/aries-framework-go/compare/1446f2c0373b23d55b26c068d9e9989fca4b8681..e9ac9108ebb0668e33f20b0cbe70fb0b87136bf9